File: LogicaC/CursoNelio/EstruturaRepetitiva/SomaImpares.c
#include <string.h>
#include <stdio.h>
#include <math.h>

void limpar_entrada()
{
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}

void ler_texto(char *buffer, int length)
{
    fgets(buffer, length, stdin);
    strtok(buffer, "\n");
}

int main()
{
    int n, i;

    printf("Quantos numeros voce vai digitar?");
    scanf("%d", &n);

    int vet[n];

    for(i=0; i < n; i++){
        printf("Digite um número: ");
        scanf("%d", &vet[i]);
    }

    printf("\n Numeros negativos: \n");

    for (i = 0; i < n; i++){
        if(vet[i] < 0){
            printf("%d\n", vet[i]);
        }
    }

    return 0;
}

File: LogicaPython/EstruturaRepetitiva/SomaImpares.py
n = int(input("Quantos números você vai digitar? "))

vet: [float] = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = float(input("Digite um número: "))

print()
print("Valores: ")

for i in range(0,n):
    print(f"{vet[i]:.1f} ", end="")

print()

soma = 0

for i in range(0,n):
    soma = soma + vet[i]

print(f"Soma: {soma:.2f}")

media = soma / n

print(f"Média: {media:.1f}")